modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/TargetContext.java

```diff
@@ -17,7 +17,12 @@
 package org.apache.synapse.transport.passthru;
 
 import org.apache.axis2.context.MessageContext;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
```

Comment:
Formatting issue here shall we fix this?

Reply:
fixed in https://github.com/sachithKay/wso2-synapse/commit/3373b68eeae9c06550355732e1f7a4fbc6c42153